app.js

```diff
@@ -1,12 +1,13 @@
-require('newrelic');
+require('newrelic'); // eslint-disable-line strict
```

Comment:
New Relic requires this line to live at Line 1 to ensure 💯 coverage. Adding `use strict` on Line 2 throws more linting errors.